airflow/providers/amazon/aws/sensors/ec2.py

```diff
@@ -62,8 +63,11 @@ def __init__(
         self.aws_conn_id = aws_conn_id
         self.region_name = region_name
 
+    @cached_property
+    def hook(self):
+        return EC2Hook(aws_conn_id=self.aws_conn_id, region_name=self.region_name)
```

Comment:
Trivial nitpick suggestion

```suggestion
        return EC2Hook(
            aws_conn_id=self.aws_conn_id, 
            region_name=self.region_name,
        )
```

Reply:
I'm usually in favor of less lines of code for more density on one screen

Reply:
That's fair. :+1:  Feel free to resolve/close this.